plugin/src/ornaments/lifting.ml

```diff
@@ -117,7 +110,7 @@ let get_direction (from_typ_app, to_typ_app) orn_kind =
   | CurryRecord ->
      not (equal Produtils.prod (first_fun from_typ_app))
   | SwapConstruct _ ->
-     failwith "not yet implemented"
+     true
```

Comment:
Explain why this is OK

Reply:
Also, in swap test, test a custom one with automatic inversion that isn't completely symmetric